src/modules/0.0.11/aws_iam/index.ts

```diff
@@ -46,13 +62,23 @@ export const AwsIamModule: Module2 = new Module2({
           for (const role of es) {
             const roleArn = await client.newRoleLin(
               role.roleName,
-              role.assumeRolePolicyDocument,
+              JSON.stringify(role.assumeRolePolicyDocument),
               role.attachedPoliciesArns ?? [],
               role.description
             );
             if (!roleArn) { // then who?
               throw new Error('should not be possible');
             }
+            try {
+              const allowEc2Service = AwsIamModule.utils.allowEc2Service(role);
+              if (allowEc2Service) {
+                await client.createInstanceProfile(role.roleName);
+                await client.attachRoleToInstanceProfile(role.roleName);
+              }
+            } catch (err: any) {
+              // Do not throw if for any reason this does not work
+              logger.warn(err.message ?? 'Error attaching instance profile on role create');
+            }
```

Comment:
Why do we not error out in this scenario?

Reply:
hmmm thought it would make sense after I did the update. But yeah I'll remove this, same with delete